src/layouts/partials/Header.astro

```diff
@@ -22,8 +22,8 @@ const { main }: { main: NavigationLink[] } = menu;
   <nav class="navbar container">
     <!-- logo -->
     <div class="order-0">
-      <Logo />
-    </div>
+      <h3> Scouts BSA Troop 150 </h1>
+    <p>Portland, OR</p></div>
```

Comment:
A little cleanup would help organize this! It should look like:
```html
<div class="order-0">
  <h3>Scouts BSA Troop 150</h3>
  <p>Portland, OR</p>
</div>
```

The changes are:
* Close with the same `</h3>` tag as the opening `<h1>` tag
* Remove the spaces at the beginning and ending of the heading line of text
* Indent the second line to be at the same indentation level
* Move the closing `</div>` to a new line at the same indentation level as the opening `<div>`